### PR TITLE
Cultures/berber

### PR DESCRIPTION
**Pull request by questionmarkexclamationpoint**
Doesn't appear to be an issue for this one.

### CODE REVIEW

**Comment by questionmarkexclamationpoint** [comment]
There's a lot of guesswork here, because there is very little information on a lot of these provinces.

Given that Berber dynasties and kingdoms ruled most of this area until the end of the game, a lot of them will remain Berber throughout.

**Comment by questionmarkexclamationpoint** [comment]
image
Cultures at the start of the game.

I was kind of unsure about the cultural makeup from Ziz down to Taghaza, so I took a guess based on later information.

**Comment by questionmarkexclamationpoint** [comment]
Also, not sure how you feel about this, but it might make more sense to make Tangier, Ceuta, Infa, and Fes become `andalusian_arabic` rather than `maghreb_arabic` later in history.

**Comment by daniloy** [comment]
Do you have access to the old ck2 plus mod? You could use it for orientation.

**Comment by daniloy** [comment]
> 
> 
> Also, not sure how you feel about this, but it might make more sense to make Tangier, Ceuta, Infa, and Fes become `andalusian_arabic` rather than `maghreb_arabic` later in history.

Fine by me.

**Comment by daniloy** [comment]
I think that Libya and Tunesia were not fully arabized at the time.

**Comment by questionmarkexclamationpoint** [comment]
Alright, I'll check out the old CK2+ for references where I'm unsure. I was trying to just use historical sources where possible. Where those don't exist I'll defer to past judgment.

**Comment by questionmarkexclamationpoint** [comment]
You're right, but they wouldn't have been any of the Berber cultures that we have implemented so I left them.

**Comment by daniloy** [comment]
> 
> 
> You're right, but they wouldn't have been any of the Berber cultures that we have implemented so I left them.

Which ones?

**Comment by questionmarkexclamationpoint** [comment]
From my reading, Libya would have likely been Houara (https://en.wikipedia.org/wiki/Houara), along with parts of The Nile. The map from the old CK2+ seems wrong in a number of places. Notably, Zenatians would have occupied much of the middle Atlas mountains. And, I'm pretty sure the Garamantians didn't stick around that long, at least not under that name.

**Comment by daniloy** [comment]
> From my reading, Libya would have likely been Houara (https://en.wikipedia.org/wiki/Houara), along with parts of The Nile. The map from the old CK2+ seems wrong in a number of places. Notably, Zenatians would have occupied much of the middle Atlas mountains. And, I'm pretty sure the Garamantians didn't stick around that long, at least not under that name.

Thanks, Anna. So, just add them if you want to, it would be great. I know that mostly the coastal areas were Arabized and less in the desert, perhaps the coasts should become Maghrebi? Anyway, your work is amazing. The guy who did history was the same who forgot to add the dynasties for the Berbers and also insisted on the Garamantians. I deleted them.

**Comment by questionmarkexclamationpoint** [comment]
Two cultures added, Houara and Kutama.

**Comment by questionmarkexclamationpoint** [comment]
And, as for becoming Arabized, Berber kingdoms ruled most of North Africa for the whole of the time period of the game. It is true that the ruling class, and eventually much of the people, would have spoken Arabic, but culturally they largely remained Berber (even in the 18th century, the North African pirates were called the Barbary pirates).

**Comment by questionmarkexclamationpoint** [comment]
For instance, [this](https://en.wikipedia.org/wiki/Zayyanid_dynasty) is the dynasty that ruled much of Algeria until the end of the game. They were Zenata.

File: CK2Plus_expanded/common/cultures/000_CK2Plus_amazigh.txt [hunk 1]
```diff
@@ -438,4 +438,158 @@ amazigh = {
 
 		modifier = default_culture_modifier
 	}
+
+	houara = {
```
Comment (questionmarkexclamationpoint) [inline CK2Plus_expanded/common/cultures/000_CK2Plus_amazigh.txt]: tbh these are straight copies of zenati.

Comment (schwarherz) [reply CK2Plus_expanded/common/cultures/000_CK2Plus_amazigh.txt]: If they're copies is there any reason to include them?

Comment (daniloy) [reply CK2Plus_expanded/common/cultures/000_CK2Plus_amazigh.txt]: we have no idea about their names, it is very hard to say to find distinguishing names among the berber tribes,

Comment (questionmarkexclamationpoint) [reply CK2Plus_expanded/common/cultures/000_CK2Plus_amazigh.txt]: Well, they're different cultures. Looking at the other cultures it seems like some of the others are fairly close to straight copies.

**Comment by LordPeter** [comment]
So, while I fully agree on the Berber thing in general (which is why we have such a culture group), there is the problem that the Kutama (and with them their most well known Zirid dynasty) were part of the Sanhaja confederation. Splitting Kutama from them without representing the other subgroups would imo be like representing Bavarian while keeping all other German cultures grouped under a generic "German" - _inconsistent_!
So I would want to have Kutama removed (and thus remain part of Sanhaja).

However, the same is not true for Houara, which from all I (and ?!) could find were a top-level confederation, which is exactly what our current Berber culture scheme represents. So Houara should be kept.

**Pull request merged**